_installers.sh

```diff
@@ -266,6 +266,8 @@ function install_rundeck {
             update_repos
         ;;
         rhel|centos|fedora)
+            _install_packages java-1.8.0-openjdk java-1.8.0-openjdk-devel
```

Comment:
This installation needs to be conditioned and the version number declared as variable

```bash
local java_version=1.8.0
if ! command -v java; then
    _install_packages "java-${java_version}-openjdk" "java-${java_version}-openjdk-devel"
fi
```

Reply:
Done 81ce793355f752f3d4d213dc019200b6281e5915